pkg/controller/reconciler/flux_controller_test.go

```diff
@@ -149,6 +149,7 @@ func setupFluxControllerTest(t *testing.T, opts setupFluxControllerTestOptions,
 		Metrics: server.Options{
 			BindAddress: "0",
 		},
+		LeaderElection: false,
```

Comment:
Is this only supposed to be one line? I saw before that there were a few changes

Reply:
I ran into some other issues with the leader election and cache sync wait logic. I checked the other test that creates a manager and saw that the difference was this line. Let's test it this way and see if we will run into issues.